rocky/rocky/templates/findings/finding_list.html

```diff
@@ -17,7 +17,7 @@ <h1>{% translate "Findings @ " %}{{ valid_time|date }}</h1>
                         Each finding relates to an object. Click a finding for additional information.
                     {% endblocktranslate %}
                 </p>
-                {{ object_type_filters }}
+                {% include "partials/show_active_filters.html" %}
```

Comment:
`object_type_filters` wasn't defined and this view also doesn't define `active_filters`. Better to remove this completely. If we want to change this to show active filters we can better do this in another PR. Before writing that code we should probably first think about how we can structure the views show we don't have to duplicate things.